packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -212,13 +226,26 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
     volumeId?: string,
     suppressEvents?: boolean
   ): void {
+    const applicableVolumeActorInfo = this._getApplicableVolumeActor(volumeId);
+
+    if (!applicableVolumeActorInfo) {
+      return;
+    }
+
+    const volumeIdToUse = applicableVolumeActorInfo.volumeId;
+
     // make sure the VOI LUT function is valid in the VOILUTFunctionType which is enum
     if (Object.values(VOILUTFunctionType).indexOf(voiLUTFunction) === -1) {
       voiLUTFunction = VOILUTFunctionType.LINEAR;
     }
     const { voiRange } = this.getProperties();
-    this.VOILUTFunction = voiLUTFunction;
     this.setVOI(voiRange, volumeId, suppressEvents);
+
+    this.updateViewportProperties(
+      volumeIdToUse,
```

Comment:
just use `applicableVolumeActorInfo.volumeId` here since you don't use volumeIdToUse anymore